miniapp/USERGUIDE.md

```diff
@@ -636,6 +636,60 @@ CoroutineScope(Dispatchers.IO).launch {
 }
 ```
 
+### Fetching internationalized Mini App Meta data
+**API Docs:** [MiniApp.getMiniAppManifest](api/com.rakuten.tech.mobile.miniapp/-mini-app/fetching-internationalized-mini-app-meta-data.html)
+
+Mini App developers able to localize text in the manifest by providing alternative text for each language which they wish to support.
+To fetch the internationalized manifest MiniApp developers need to add the following attributes in their MiniApp.
+
+```json
+{
+    "reqPermissions": [
+        {
+            "name": "rakuten.miniapp.user.USER_NAME",
+            "reason": "__MANIFEST_userNameReason__"
+        }
+    ],
+    "optPermissions": [
+        {
+          "name": "rakuten.miniapp.user.CONTACT_LIST",
+          "reason": "__MANIFEST_contactList__"
+        }
+      ],
+    "customMetaData": {
+        "exampleKey": "__MANIFEST_exampleText__",
+        "notTranslatedKey": "This key has no translations."
+    }
+}
+
+// _locale/default/manifest.json:
+{
+  "userNameReason": "Default text.",
+  "contactList": "Default contactList text.",
+  "exampleText": "Default text."
+}
+
+// _locale/ja/manifest.json:
+{
+  "userNameReason": "Japanese text.",
+  "contactList": "Japanese contactList text.",
+  "exampleText": "Japanese text."
+}
```

Comment:
Not sure if we need to mention this in the Userguide. Because SDK developers won't be using these files. They will use just the API's to get the localized description/reason.

Reply:
@rleojoseph Yeah. As we included the manifest json format previously, so I thought it's good to mention the information about `_locale` too. But, since we will have separate guide for MiniApp developers, so we can skip this. 

@corycaywood What do you think? 🤔

Reply:
I'm not sure - maybe it makes more sense to put this in the JS SDK documentation?

Reply:
Okay, I have updated here. Thank you guys 👍